src/components/response/ResponseBlock.tsx

```diff
@@ -62,6 +66,26 @@ export default function ResponseBlock({
 
   const showNextBtn = location === (configInUse?.nextButtonLocation || 'belowStimulus');
 
+  const nextButtonDisableTime = configInUse?.nextButtonDisableTime;
+  const nextButtonEnableTime = configInUse?.nextButtonEnableTime || 0;
+  const [timer, setTimer] = useState<number | undefined>(undefined);
+  // Start a timer on first render, update timer every 100ms
+  useEffect(() => {
+    let time = 0;
+    const interval = setInterval(() => {
+      time += 100;
+      setTimer(time);
+    }, 100);
```

Comment:
Is it necessary to be so fine-grained? Maybe using 500ms instead? Honestly not sure if this will ever make a difference, but the less renders the better, IMO

Reply:
Changed to 500ms